pkgs/servers/search/manticoresearch/default.nix

```diff
@@ -60,6 +60,11 @@ stdenv.mkDerivation rec {
     "-DWITH_GALERA=0"
   ];
 
+  postFixup = ''
+    # Remove references to nix store in dist config file
+    sed -i 's|/nix/store/.*/var|/var|' $out/etc/manticoresearch/manticore.conf
+  '';
+
```

Comment:
ah, thanks - I wondered what a good way of setting sane paths in the default config was.